src/request_tracing.cpp

```diff
@@ -38,42 +38,45 @@ bool should_delegate(ngx_http_request_t *request,
   return loc_conf->sampling_delegation_enabled &&
          loc_conf->allow_sampling_delegation_in_subrequests;
 }
+
+std::optional<std::string> set_if_exists(ngx_http_complex_value_t *conf,
```

Comment:
nit: this is not setting anything so i would change the name of the function

Reply:
addressed in 248a4c32bc084b3b955c31baa5bcb43dfa27c295